demo/form/pta/playground

```diff
@@ -1,18 +1,19 @@
-const settings = {
-  enableDraft: true,
-  draftId: 'CochesnetInsert-draft',
-  authToken: 'dXNlcl9pZD0xNzAwMDU0JmF1dGhlbnRpY2F0aW9uX3Rva2VuPWZhMmE5ZTg1LTY3ZTYtNDU0NC1hNTNmLWQ1NjIwZTMyYjZmNyZzZXNzaW9uX2lkPTM1cmNlNG5pZ2VsaHRyM2FseW54YmllMg==',
-  appTheme: 'coches',
-  language: 'es',
-  formUrl: 'https://webformbuilderpre.scmspain.com/index.html',
-  redirectOnSuccessUrl: 'http://pre.cochesnet.es',
-  redirectOnErrorUrl: 'http://pre.cochesnet.es/mvc/property/NextStep?ai=${id}&mode=1&ot=4',
-  adsUrl: 'https://ptaadinsertion-classifiedads.global-pre.spain.schibsted.io/cochesnet/ads',
-  imagesUrl: 'https://ptaphotouploader-classifiedads.global-pre.spain.schibsted.io/cochesnet/photos',
-  appID: 'cochesnet-insert-car'
-}
-
+let settings = {
```

Comment:
this data isn't confidential, right?

Reply:
authToken expires, but anyway, i will hide it. Thanks!